Integrations/src/main/java/io/deephaven/integrations/learn/Scatterer.java

```diff
@@ -0,0 +1,54 @@
+package io.deephaven.integrations.learn;
+
+import io.deephaven.base.verify.Require;
+import io.deephaven.datastructures.util.CollectionUtil;
+import org.jpy.PyObject;
+
+import java.util.ArrayList;
+
+/**
+ * Scatterer applies scatter functions to the result of a deferred calculation and feeds the scattered data back into a table.
+ */
+public class Scatterer {
+
+    private final Output[] outputs;
+
+    /**
+     * Creates a new Scatterer.
+     *
+     * @param outputs array of Outputs that determine how data will be scattered back to the table.
+     */
+    public Scatterer(Output[] outputs) {
+
+        Require.neqNull(outputs, "outputs");
```

Comment:
Need to require that all slots are not null.  I think there is a require for this.